backend/src/routes/upload.ts

```diff
@@ -53,6 +53,11 @@ export class UploadRoutes {
     let uploadRepo: Repository<InstrumentUpload | ModelUpload>
     let productRepo: Repository<RegularFile | ModelFile>
 
+    const send200WithCustomMessage = () => {
```

Comment:
Tarvitseeko tämän olla funktio? Voisitko vain määrittää `msg`in täällä alussa ja sitten aina kutsua lopuksi `res.send(msg)`. TIetty niin, että `msg` olisi paremmin nimetty.

Reply:
Joo ihan turha funktio, mitäköhän mä ajattelin tässä.